frontend/src/features/documents/commonStyles.tsx

```diff
@@ -0,0 +1,36 @@
+import { Scrollable } from 'components/common/Scrollable/Scrollable';
+import styled from 'styled-components';
+
+export const StyledGreySection = styled.div`
+  padding: 1rem;
+  background-color: ${({ theme }) => theme.css.filterBackgroundColor};
+`;
+
+export const StyledH2 = styled.h2`
```

Comment:
I'm always a bit worried when I see new h* definitions. Do our existing centralized h2, h3 definitions not work for this page? If not we may want to discuss that with Bobbi, as we are trying to make our headers consistent.

Reply:
Not 100% sure, but given this is a bugfix I rather not make that change here. As it is, is just a refactor of what was already used in the styles.